.circleci/config.yml

```diff
@@ -260,7 +260,7 @@ jobs:
       - run:
           name: Check vendored schema for upstream updates
           command: |
-            bin/update-schema.sh HEAD
+            bin/update-schema.sh main
```

Comment:
What's this about?

Reply:
should have left a comment: GitHub changed _something_ and now `HEAD` is not valid anymore in direct URLs to files: https://app.circleci.com/pipelines/github/mozilla/glean/9214/workflows/436eec8e-cab6-405e-b2f6-f1739852ed03/jobs/170397

Reply:
Yeah, looks like it was a bug that's being fixed :)